anubis-management-api/src/tenants/operations.py

```diff
@@ -32,6 +36,25 @@ def create_tenant(db: Session, tenant: schemas.TenantCreate):
     db.add(default_service_path)
     db.commit()
     db.refresh(new_tenant)
+    # creating default policy
+    # TODO: This will eventually read from a non yaml file
+    with open(os.environ.get("DEFAULT_POLICIES_CONFIG_FILE", '../../config/opa-service/default_policies.yml'), 'r') as file:
+        default_policies = yaml.load(file, Loader=yaml.FullLoader)
+        for p in default_policies["acl:Authorization"]:
+            if p["acl:accessTo"]["value"] == "default":
+                policy = policy_schemas.PolicyCreate(
```

Comment:
probably you don need this if.

access_to is anyhow default

the only different is `resource_type="*",`

which could be auto-set during policy creation when access_to = default
or throw a validation error if the value is not valid

Reply:
You need this if because the default_policies.yml file can also contain policies that aren't marked as "default", meaning they refer to specific entities and types.

Reply:
yes, but again, if you do policy validation, this if is not needed.

Reply:
But this is calling the method to create a policy directly, it's not going through the schema that does the validation. So I'm doing the check here

Reply:
not very clean though :/

Reply:
Wait actually, it is checked by the schema when calling that method, so the validation happens. Removing the if then, it will throw a validation error if it's not set correctly.